frontend/lib/framework/resolution-error-classes-list.cpp

```diff
@@ -550,6 +550,32 @@ void ErrorUnsupportedAsIdent::write(ErrorWriterBase& wr) const {
   wr.code(expectedIdentifier, { expectedIdentifier });
 }
 
+void ErrorUseImportMultiplyDefined::write(ErrorWriterBase& wr) const {
+  auto symbolName = std::get<UniqueString>(info);
+  auto firstOccurrence = std::get<1>(info);
+  auto secondOccurrence = std::get<2>(info);
+
+  wr.heading(kind_, type_, secondOccurrence, "identfier '",
```

Comment:
spelling error here

also I think it might be better not to even say "identifier" here as it is jargon. IMO it could just say "'a' is multiply defined" without loss of clarity.